tests/manage/storageclass/test_create_multiple_sc_with_same_pool_name.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+import logging
+from tests import helpers
+from ocs_ci.ocs import constants
+from ocs_ci.ocs import defaults
+from ocs_ci.ocs.resources.pod import get_fio_rw_iops
+from ocs_ci.framework.testlib import ManageTest, tier1
+from tests.fixtures import (
+    create_ceph_block_pool,
+    create_rbd_secret, create_cephfs_secret,
+    create_project
+)
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def resources(request):
+    """
+       Delete the resources created during the test
+       Returns:
+           tuple: empty lists of resources
+       """
+    pods, pvcs, storageclasses = ([] for i in range(3))
+
+    def finalizer():
+        """
+            Delete the resources created during the test
+            """
+        for resource_type in pods, pvcs, storageclasses:
+            for resource in resource_type:
+                resource.delete()
+                resource.ocp.wait_for_delete(resource.name)
```

Comment:
in case it will fail in this wait_for_delete method it will raise TimeoutError exception and you will not delete the rest of the resources right?

You can improve when you will surround this operation by try catch, and in case of error you will store the errors in some list, and at the end of finalizer you can check if this is not empty and raise the exception and you will be sure that you at least tried to delete all the resources.

Reply:
Done